modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -191,6 +191,16 @@ public void flatten(Swagger swagger) {
         }
     }
 
+    private void fixStringModel(ModelImpl m) {
```

Comment:
@JFCote it would be nice to add a 1-liner or a proper docstring explaining what this function does.